drivers/gles3/storage/particles_storage.cpp

```diff
@@ -271,13 +271,13 @@ void ParticlesStorage::particles_set_fractional_delta(RID p_particles, bool p_en
 
 void ParticlesStorage::particles_set_trails(RID p_particles, bool p_enable, double p_length) {
 	if (p_enable) {
-		WARN_PRINT_ONCE("The OpenGL 3 renderer does not support particle trails");
+		WARN_PRINT_ONCE_ED("The GL Compatibility rendering backend does not support particle trails");
 	}
 }
 
 void ParticlesStorage::particles_set_trail_bind_poses(RID p_particles, const Vector<Transform3D> &p_bind_poses) {
 	if (p_bind_poses.size() != 0) {
-		WARN_PRINT_ONCE("The OpenGL 3 renderer does not support particle trails");
+		WARN_PRINT_ONCE_ED("The GL Compatibility rendering backend does not support particle trails");
```

Comment:
Stylistic change for consistency with other messages::

```suggestion
		WARN_PRINT_ONCE_ED("The GL Compatibility rendering backend does not support particle trails.");
```